Library/Formula/slimerjs.rb

```diff
@@ -7,10 +8,10 @@ def self.firefox_installation
     paths.find { |p| File.exist? File.expand_path(p) }
   end
 
-  satisfy { FirefoxDependency.firefox_installation }
+  satisfy { FirefoxDependency.firefox_installation || Formula["xulrunner"].installed? }
```

Comment:
I didn't know if we wanted to always prefer Firefox if it was available, or always prefer our ` xulrunner ` formula if available. Will flip them around :+1: